service/authorization/authorization.go

```diff
@@ -425,16 +425,16 @@ func makeValsByFqnsLookup(attrs []*policy.Attribute, subMapsByVal map[string][]*
 	return fqnAttrVals
 }
 
-// makeScopeMap creates a map of attribute value FQNs.
-func makeScopeMap(scope *authorization.ResourceAttribute) map[string]bool {
+// MakeScopeMap creates a map of attribute value FQNs.
+func MakeScopeMap(scope *authorization.ResourceAttribute) map[string]bool {
```

Comment:
@suchak1  is there a reason for this to be public?

Reply:
reverted